src/main/java/frc/robot/Robot.java

```diff
@@ -273,6 +274,28 @@ public void robotInit() {
                     elevator.setExtensionCmd(() -> ElevatorSubsystem.AMP_EXTENSION_METERS))
                 .withTimeout(0.75));
     controller.rightBumper().whileTrue(swerve.stopWithXCmd());
+
+    // ---New climb----
+    operator
```

Comment:
check the current climb button bindings and use those. should be operator right bumper + right trigger to start cliimb, driver y to pull up, and driver y again to trap